.github/workflows/ci.yml

```
@@ -0,0 +1,238 @@
+"""
```

actionlint (1.7.7)

workflow is scalar node but mapping node is expected

(syntax-check)